R/markdown.R

```diff
@@ -15,44 +14,15 @@ markdown <- function(text, tag = NULL, sections = FALSE) {
   markdown_pass2(escaped_text, tag = tag, sections = sections)
 }
 
-#' Expand the embedded inline code
+#' Expand R code
```

Comment:
I think this needs to be left substantially as is as it serves as a convenient regression test as well as documentation; I'd suggest just adding a mention of the vignette.